README.md

```diff
@@ -78,7 +78,7 @@ The Logisim-evolution.app is not code-signed.
 
 When launching the application for the first time, you will have to start it via the "Open" entry in the
 application icon's context menu in the macOS Finder. This is either done by clicking the application
-icon with the right mouse button or holding down <kbd>CTRL<kbd> leu while clicking the icon with the
+icon with the right mouse button or holding down <kbd>CTRL</kbd> leu while clicking the icon with the
```

Comment:
You fixed the closing tag, but you didn't delete the "leu". I should have been more clear that there were two problems.

It occurs to me that "leu" may be a mistyped "key". If you want to add "key" there, add "the" before CTRL. I would just delete the "leu".

Reply:
I believe I for some reason too "leu" as part of phrase like "in lieu" :) Fixed.